softioc/cothread_dispatcher.py

```diff
@@ -19,6 +19,11 @@ def __init__(self, dispatcher = None):
         else:
             self.__dispatcher = dispatcher
 
+        def wait_for_quit():
+            cothread.WaitForQuit()
+
+        self.wait_for_quit = wait_for_quit
```

Comment:
Actually this can be more simply written as
```python
self.wait_for_quit = cothread.WaitForQuit
```
No need to declare an extra function!

Reply:
You're right, I'll make the change. 
Thank you for your feedback